Web_Development/Javascript/Topics/5.Syntax/1.Data_Types/1.Data_Types_In_JavaScript.md

```diff
@@ -0,0 +1,103 @@
+# Data Types in JavaScript
+
+Just like any other programming language, JavaScript has data types too, however they might be different in their behaviour and properties unlike data types in other programming languages.
+
+## Primitive Types
+---
+JavaScript has 5 main primitive types :
+
+- String
+- Boolean
+- Number
+- Null and Undefined
+- BigInt
+
+*Note: Objects in JavaScript are a collection of properties and is not considered under Primitive types*
+
+## String Type
+---
+String type is used to represent textual data / information in JavaScript. It is a set of elements where each element occupies a position in the String. The first element is at 0th index, the next at index 1, and so on. The length of a String is the number of elements in it.
+Single quotes or double quotes may be used to assign a value to a string variable.
+
+Example:
+
+```
+    let str = 'javascript';
+```
+
+*Note: In JavaScript Strings are immutable, that means once they are created we cannot modify the same*
+
+## Boolean Type
+---
+Boolean type in JavaScript represents two values, logical true (or 1) or logical false (or 0)
+
+Example:
+
+```
+    let isGameOver = true;
+    isGameOver = 0;
+```
+
+## Number Type
+---
+JavaScript has only one number type. Decimals, Integers, Whole numbers all are represented using a single type called Number. Unlike, other programming languages, JavaScript doesn't have different data types for different types of numbers.
+
+Example:
+
+```
+    let x = 12.34;
+    let y = 123;
+    let negativeNumber = -34;
+```
```

Comment:
No need to indent the code. Also, use "\`\`\`js" instead of "\`\`\`" to begin with.

````
```js
let x = 12.34;
let y = 123;
let negativeNumber = -34;
```
````